### DISCUSSION TITLE
Strava OAuth Support?

### DISCUSSION THREAD

**Discussion by jahamed** [supabase/supabase#15381] [Feature Requests]
Are there any plans to add Strava as an authentication provider? Next Auth currently supports it: https://next-auth.js.org/providers/strava 

Need to be authenticated to do most things with the Strava API, so having it as a login for users for a fitness app would make sense.

**Comment by Ugoyo** [comment]
It would be great to have Strava as Auth Providers directly integrated into Supabase.

**Comment by vinhlh** [comment]
+1
